.azure/pipelines/ci-public.yml

```diff
@@ -628,43 +628,44 @@ stages:
-      parameters:
-        jobName: Local_Windows
-        jobDisplayName: 'Test: Windows local development validation'
-        agentOs: Windows
-        isAzDOTestingJob: true
-        timeoutInMinutes: 240
-        steps:
-        - script: git submodule update --init
-          displayName: Update submodules
-        - script: ./restore.cmd
-          displayName: Run restore.cmd
-        - powershell: ./eng/build.ps1 -all -noBuildJava -noBuildNodeJS "-WarnAsError:false"
-          displayName: Build (No NodeJS)
-        - script: npm run build
-          displayName: Build JS
-        - script: ./eng/build.cmd -all -noBuildJava -pack -c Debug
-          displayName: Build (Debug)
-        - script: ./eng/build.cmd -all -noBuildJava -pack -c Release
-          displayName: Build (Release)
-        - script: ./src/ProjectTemplates/build.cmd
-                  -test
-                  -NoRestore
-                  -NoBuild
-                  -NoBuildDeps
-                  -configuration Release
-                  -bl
-          displayName: Run project template tests
-        - powershell: . ./activate.ps1; ./src/ProjectTemplates/scripts/Run-BlazorWeb-Locally.ps1 -Verbose
-          displayName: Run Blazor web app test script
-
-        artifacts:
-        - name: Local_Windows_x64_Logs_Attempt_$(System.JobAttempt)
-          path: artifacts/log/
-          publishOnError: true
-          includeForks: true
+    # - template: jobs/default-build.yml
+    #   parameters:
+    #     jobName: Local_Windows
+    #     jobDisplayName: 'Test: Windows local development validation'
+    #     agentOs: Windows
+    #     isAzDOTestingJob: true
+    #     timeoutInMinutes: 240
+    #     steps:
+    #     - script: git submodule update --init
+    #       displayName: Update submodules
+    #     - script: ./restore.cmd
+    #       displayName: Run restore.cmd
+    #     - powershell: ./eng/build.ps1 -all -noBuildJava -noBuildNodeJS "-WarnAsError:false"
+    #       displayName: Build (No NodeJS)
+    #     - script: npm run build
+    #       displayName: Build JS
+    #     - script: ./eng/build.cmd -all -noBuildJava -pack -c Debug
+    #       displayName: Build (Debug)
+    #     - script: ./eng/build.cmd -all -noBuildJava -pack -c Release
+    #       displayName: Build (Release)
+    #     - script: ./src/ProjectTemplates/build.cmd
+    #               -test
+    #               -NoRestore
+    #               -NoBuild
+    #               -NoBuildDeps
+    #               -configuration Release
+    #               -bl
+    #       displayName: Run project template tests
+    #     - powershell: . ./activate.ps1; ./src/ProjectTemplates/scripts/Run-BlazorWeb-Locally.ps1 -Verbose
+    #       displayName: Run Blazor web app test script
+
+    #     artifacts:
+    #     - name: Local_Windows_x64_Logs_Attempt_$(System.JobAttempt)
+    #       path: artifacts/log/
+    #       publishOnError: true
+    #       includeForks: true
```

Comment:
I think there's a better way to do it (marking the pipeline as not reuqired or something like that) @akoeplinger will know better.

Reply:
There's `condition:` param, we can pass `false` there